.goreleaser.yaml

```diff
@@ -0,0 +1,81 @@
+project_name: apm-lambda-extension
+
+before:
+  hooks:
+    # test
+    - 'sh -c "go test extension/*.go -v"'
+    # check-licenses
+    - go run github.com/elastic/go-licenser@v0.4.0 -d -exclude tf -exclude testing .
+    - go run github.com/elastic/go-licenser@v0.4.0 -d -exclude tf -exclude testing -ext .java .
+    - go run github.com/elastic/go-licenser@v0.4.0 -d -exclude tf -exclude testing -ext .js .
+    # NOTICE.txt
+    - ./scripts/notice.sh
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w
+    binary: "extensions/{{ .ProjectName }}"
+
+archives:
+  - id: zip
+    format: zip
+    name_template: |-
+      {{ .Tag }}-{{ .Os }}-{{ .Arch }}
+    files:
+      - NOTICE.txt
+      - dependencies.asciidoc
+    rlcp: true # https://goreleaser.com/deprecations/#archivesrlcp
+
+dockers:
+
+  - &default-docker-image
+    id: linux-amd64-image
+    use: buildx
+    goos: linux
+    goarch: amd64
+    image_templates:
+      - 'docker.elastic.co/observability/{{ .ProjectName }}-x86_64:{{ trimprefix .Tag "v" }}'
+      - "docker.elastic.co/observability/{{ .ProjectName }}-x86_64:latest"
+    build_flag_templates:
+      - "--platform=linux/amd64"
```

Comment:
NIT: What do you think to leverage the build metadata with the opencontainers ones?

```
    - "--label=org.opencontainers.image.created={{.Date}}"
    - "--label=org.opencontainers.image.title={{.ProjectName}}"
    - "--label=org.opencontainers.image.revision={{.FullCommit}}"
    - "--label=org.opencontainers.image.version={{.Version}}"
```

See https://goreleaser.com/customization/docker/#applying-docker-build-flags

Reply:
https://github.com/elastic/apm-aws-lambda/pull/362/commits/f3b775a20760b2848b539adcc00cfd9b85bf4f08